src/WebJobs.Script/Workers/Rpc/RpcWorkerConstants.cs

```diff
@@ -63,5 +63,8 @@ public static class RpcWorkerConstants
         public const string PythonThreadpoolThreadCount = "PYTHON_THREADPOOL_THREAD_COUNT";
         public const string PSWorkerInProcConcurrencyUpperBound = "PSWorkerInProcConcurrencyUpperBound";
         public const string DefaultConcurrencyLimit = "1000";
+
+        // Metric events
+        public const string WorkerMetadataMetricEvent = "WorkerMetadata";
```

Comment:
Is there a style guide we follow for event names? would something like `host.workerinitresponse.workermetadata` be a more fitting event name?

Reply:
For reference, here are what some of the events look like in the table:

![Screen Shot 2022-06-17 at 11 34 39 AM](https://user-images.githubusercontent.com/2198905/174357708-bd19a18f-8e83-4d59-bf43-e1e6e3f59f49.png)